docs_src/getting-started/Ch-GettingStartedSDK-Go.md

```diff
@@ -12,30 +12,30 @@ See the [Getting Started - Go Developers](Ch-GettingStartedGoDevelopers.md) guid
 Follow these steps to create a folder on your file system,
 download the [Device SDK](../../microservices/device/sdk/Ch-DeviceSDK), and get the GoLang device service SDK on your system.
 
-1.  Create a collection of nested folders, `~/go/src/github.com/edgexfoundry` on your file system. This folder will hold your new Device Service. In Linux, create a directory with a single mkdir command
+1.  Create a collection of nested folders, `~/edgexfoundry` on your file system. This folder will hold your new Device Service. In Linux, create a directory with a single mkdir command
     ``` bash
-    mkdir -p ~/go/src/github.com/edgexfoundry
+    mkdir -p ~/edgexfoundry
     ```
 
 2. In a terminal window, change directories to the folder just created and pull down the SDK in Go with the commands as shown.
     ``` bash
-    cd ~/go/src/github.com/edgexfoundry
-    git clone --depth 1 --branch v1.2.2 https://github.com/edgexfoundry/device-sdk-go.git
+    cd ~/edgexfoundry
+    git clone --depth 1 --branch v2.0.0 https://github.com/edgexfoundry/device-sdk-go.git
     ```
 
     ![image](EdgeX_GettingStartedSDKClone.png)
 
     !!! Note
-        The clone command above has you pull v1.2.2 of the Go SDK which is the version associated to Geneva.  There are later releases of EdgeX.  While backward compatible, it is always a good idea to pull and use the latest version associated with the major version of EdgeX you are using.  You may want to check for the latest released version by going to https://github.com/edgexfoundry/device-sdk-go and look for the latest release.
+        The clone command above has you pull v2.0.0 of the Go SDK which is the version associated to Ireland.  There are later releases of EdgeX, it is always a good idea to pull and use the latest version associated with the major version of EdgeX you are using.  You may want to check for the latest released version by going to https://github.com/edgexfoundry/device-sdk-go and look for the latest release.
```

Comment:
There are later releases of EdgeX, **and** it is always a good idea to pull and use the latest version associated with the major version of EdgeX you are using.